addons/website/static/src/snippets/s_tabs/003.scss

```
@@ -0,0 +1,97 @@
+// Tabs
+.s_tabs_common {
+
+    --tabs-bg-color: #{$gray-100};
+    --tabs-link-color: #{$body-color};
+
+    .nav-underline {
+        --nav-underline-link-active-color:currentColor;
+    }
+
+    .s_tabs_content {
+        &.s_tabs_slide_up, &.s_tabs_slide_down, &.s_tabs_slide_left, &.s_tabs_slide_right {
+            > .tab-pane.fade {
+                transition: all 0.2s;
+            }
+            > .tab-pane.fade.show {
+                transform: translateX(0rem) translateY(0rem);
+            }
+        }
+        &.s_tabs_slide_up > .tab-pane.fade {
+            transform: translateY(-1rem);
+        }
+        &.s_tabs_slide_down > .tab-pane.fade {
+            transform: translateY(1rem);
+        }
+        &.s_tabs_slide_left > .tab-pane.fade {
+            transform: translateX(-1rem);
+        }
+        &.s_tabs_slide_right > .tab-pane.fade {
+            transform: translateX(1rem);
+        }
+    }
+
+    .s_tabs_main {
+        --card-cap-bg: initial;
+
+        .nav-tabs{
+            --nav-tabs-link-hover-border-color: transparent;
+        }
+
+        .s_tabs_nav:has(.nav-tabs), .s_tabs_nav:has(.nav-buttons) {
+            background: var(--tabs-bg-color, $gray-100);
+
+            .nav .nav-item a.nav-link{
+                color: var(--tabs-link-color, $body-color);
+            }
+        }
+
+        .nav-tabs a.active {
+            color: $body-color;
+        }
+
+        .nav-underline {
+            border-bottom: 1px solid $border-color;
+
+            .nav-link {
+                margin-bottom: -1px;
```

It's unlikely that the default border thickness will ever change.. but who knows, maybe one day we'll link it to an option that the user can customize globally (?)... also, good practice and code readability ;)

```suggestion
            border-bottom: $border-width solid $border-color;

            .nav-link {
                margin-bottom: $border-width * -1;
```